roles/junit2json/meta/argument_specs.yml

```diff
@@ -0,0 +1,47 @@
+---
+# arguments spec file for redhatci.ocp.junit2json role
+argument_specs:
+  main:
+    short_description: Main entry point for role redhatci.ocp.junit2json
+    description: |
+      The resulting JSON file(s) are of the same structure for all the teams' and CI systems and used later to be sent to the data collection system.
+      This is the main entrypoint for the role `redhatci.ocp.junit2json`.
+      Converts XMLs into JSON, if variable `junit2json_do_merge` is `true`, multiple XMLs are merged into one XML file.
+      New filename(s) is(are) based on the old ones and stored in global variable `global_json_reports_list`.
+    author:
+      - Max Kovgan
+    options:
```

Comment:
What about `global_json_reports_list`? This is in defaults file. Should it appear here?